packages/base/cards-grid.gts

```diff
@@ -464,7 +441,7 @@ class Isolated extends Component<typeof CardsGrid> {
       `${baseRealm.url}cards-grid/CardsGrid`,
     ];
 
-    this.filters.splice(2, this.filters.length);
+    this.filters.splice(1, this.filters.length);
```

Comment:
we're just removing the first item from the filter here? perhaps a clearer way to write this would be:
```
this.filters.shift();
```
could you also add a comment why we are removing the first item from the array?

Reply:
We're removing all items except the first one. However, I added the comment.